### DISCUSSION TITLE
Error: Milvus is not ready yet. in attu but the proxy pod show success logs

### DISCUSSION THREAD

**Discussion by jubingc** [milvus-io/milvus#36669] [Q&A and General discussion]
Attu has been installed for a few weeks, it always worked until yesterday. The following logs are from attu pod:
```
POST /api/v1/milvus/connect 500 Error: Milvus is not ready yet.
POST 500 /api/v1/milvus/connect 88.064 ms @ Tue, 08 Oct 2024 03:03:12 GMT ::ffff:10.48.154.214 from undefined Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/
```

While proxy pod shows success logs in my understanding
```
{"level":"INFO","time":"2024/10/08 03:03:12.150 +00:00","caller":"proxy/impl.go:5868","message":"connect received","traceID":"28024a7492dc25e68b3217562103692f","sdk_type":
"nodejs","sdk_version":"2.4.3","local_time":"2024-10-08 03:03:12.147","user":"admin","host":"","db":"default"}
{"level":"INFO","time":"2024/10/08 03:03:12.152 +00:00","caller":"connection/manager.go:106","message":"client register","traceID":"28024a7492dc25e68b3217562103692f","sdk_
type":"nodejs","sdk_version":"2.4.3","local_time":"2024-10-08 03:03:12.147","user":"admin","host":"","identifier":453078310486278148,"last_active_time":"2024/10/08 03:03:1
2.151 +00:00"}
```

milvus_cli is able to log in to the milvus server. All pods are in a ready state. Both attu and proxy pods have been restarted, but the issue still exists.

Can I have some suggestions on what I can check next? Thanks!

**Comment by yhmo** [comment]
Error code 500 is a server-side error, is there any WARN in the log of the proxy?
Which version of the attu? Which version of the milvus?

**Reply by jubingc** [reply]
No WARN or ERROR in logs of the proxy. The weird thing is that attu worked previously and there was no recent upgrade.

Milvus version is 2.4.9 deployed by helm chart 4.2.8. Attu version is 2.4.7 deployed by the same helm chart version.

**Reply by yhmo** [reply]
There is an issue: https://github.com/milvus-io/milvus/issues/36337
It could be related to the recent change in the health check api. It would be helpful if you could provide the full log of the milvus.

**Reply by jubingc** [reply]
Thanks. I post logs to the issue.

**Comment by a28124791** [comment]
check space

**Reply by jubingc** [reply]
What space?

**Reply by a28124791** [reply]
disk space
